mobile/security.html

```diff
@@ -50,7 +50,7 @@ <h2>Technologies in progress</h2>
           <p>To facilitate the authentication of users to on-line services, the <a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a> is proposing a <a data-featureid="credential-management">Credential Management API</a> that lets developers interact more seamless with user-agent-managed credentials.</p>
         </div>
 
-        <p data-feature="Authentication">Building on the multi-factor authentication work of the FIDO Alliance, <a data-featureid="webauthn">Secure Authentication</a> aims to standardize multi-factor authentication for the Web, using the combination of "something you have" with "something you know," so that hacking a password database is no longer sufficient to hijack user accounts.</p>
+        <p data-feature="Authentication">Building on the password-less and multi-factor authentication work of the FIDO Alliance, <a data-featureid="webauthn">Web Authentication</a> aims to standardize strong authentication for the Web, using the combination of "something you have" with "something you know," so that hacking a password database is no longer sufficient to hijack user accounts.</p>
```

Comment:
I believe it's "passwordless" and not "password-less" (a quick Web search seems to confirm that).

Also, while the authenticator in Web Authentication is "something you have", it may be coupled with "something you know" and/or "something you are" (as [described in the spec](https://w3c.github.io/webauthn/#sctn-authentication-factor-capability)), so I suggest to use that opportunity to reformulate the end of the sentence into:

[[
using a combination of "something you have" (e.g. an authentication key) coupled with "something you know" (e.g. a PIN code) and/or "something you are" (e.g. a fingerprint), so that...
]]

Reply:
Fixed. Thank you!